.claude/agents/integration-qa.md

```
@@ -0,0 +1,153 @@
+
+For streaming endpoints, verify frontend fetch and backend StreamingResponse are aligned:
+
+```bash
+# Backend: check streaming response format
+grep -r "StreamingResponse\|EventSourceResponse\|text/event-stream" backend/src/routes/
+
+# Frontend: check SSE parsing
+grep -r "data: \|event:\|EventSource\|getReader" frontend/src/
+```
+
+**Check**: Response uses `text/event-stream` content type. Frontend parses `data: ` prefixed lines. `[DONE]` sentinel is handled.
```

markdownlint-cli2 (0.21.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)